mocks/Client.go

```diff
@@ -162,23 +228,30 @@ func (_m *Client) DescribeWorkflowExecution(ctx context.Context, workflowID stri
 }
 
 // ExecuteWorkflow provides a mock function with given fields: ctx, options, workflow, args
-func (_m *Client) ExecuteWorkflow(ctx context.Context, options client.StartWorkflowOptions, workflow interface{}, args ...interface{}) (client.WorkflowRun, error) {
```

Comment:
All these `internal.*`  need  to be replaced by  the appropriate exposed type.

Reply:
@Quinn-With-Two-Ns 
thanks for review.
Actually `internal.WorkflowRun` is equivalent to `client.WorkflowRun`. FYI.
https://github.com/temporalio/sdk-go/blob/41d2288301cc205de56e1d7f5841e56edcb5f40d/client/client.go#L100

I will still make the change

Reply:
It requires manual approval